.github/workflows/bazel.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
 
       - name: run tests
         run: bazel test //bazel/gtest:hello_test
+        continue-on-error: true
```

Comment:
We need a story for how we handle the action crashing. If it crashes (and no tests run or stale test files exist) then the upload + quarantine step will wrongfully succeed.

Reply:
Agreed. This is just a happy-path test right now to see green checks and unblock some other work.

Would it be sufficient to add a check in the cli that if `run` is not set and no new output files matching the blob are found, we exit_failure?

Reply:
It may be useful to have a pre step that helps us track what happened between job start and upload (so that we can figure out what happened). That's one idea, but can you put together a more concrete proposal on the flows here?

Reply:
>  If it crashes (and no tests run or stale test files exist), then the upload + quarantine step will wrongfully succeed.

It should be pretty straightforward to fail if no files are found and the uploader is running with the `quaranting` flag, and I think it's a predictable behavior.

I am not sure about stale files 🤔. We could ask users to set a timestamp before running the test step to exclude files older than the test start, but  I will be hesitant to add anything more involved.